src/crypto/oaes_lib.c

```diff
@@ -1273,7 +1273,8 @@ OAES_RET oaes_encrypt( OAES_CTX * ctx,
 		
 		// insert pad
 		for( _j = 0; _j < OAES_BLOCK_SIZE - _block_size; _j++ )
-			_block[ _block_size + _j ] = _j + 1;
+			uint8_t offset = _j;
```

Comment:
needs to be in a curly brace block as this for loop without one will be just the first line.

also iirc, there actually may be a bad side effect in the old code that is required here.

Reply:
also, i dont think this has the same logic as before.